pkg/password/mod.ts

```diff
@@ -1,7 +1,7 @@
-// import { genSalt, hash, verify } from 'scrypt';
-import { argon2id, hash, verify } from 'argon2';
+import { hash, verify } from 'argon2';
 
 export type EncodedPassword = string;
+const argon2id = 2;
```

Comment:
```suggestion
const ARGON2_ID = 2;
```
定数なので `SCREAMING_SNAKE_CASE` にしておきたいです.